### ISSUE TITLE
Elaborate on any vague known issue

### ISSUE THREAD

**Issue by phistuck** [Fyrd/caniuse#2488]
Some known issues are vaguely explained and it is hard to determine or reproduce the actual issue in order to create an issue in the respective issue tracker of the browser.

For example -
The known issue that involves `will-change` and `background-attachment` - http://caniuse.com/#feat=background-attachment
See this discussion for background -
https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/LJvllptlDdM

It would be great and extremely helpful if you could at least add a JSFiddle link (or similar, as appropriate) to any known issue you mention, or the report in the wild that you encountered that mentions the issue. Or better yet, a link to a specific issue in the respective issue tracker of the browser or rendering engine.

Thank you!

**Comment by Fyrd** [comment]
Yeah, I fully agree with this. Most of the "known issues" come from the many submissions I get via the Feedback form and if they relate to random bugs I historically haven't always had the time to verify them. But recently I've come to the same conclusion and have only been adding new ones that include links to either bug reports or fiddles, etc.

For the existing issues of this type, do you think the site would be better served removing them if they don't include links?

**Comment by phistuck** [comment]
Good question... perhaps add a "(unverified, can you verify?)" link (to a wiki page, say) or something like that and it might trigger someone to verify and add an issue number...
